plugin.video.putio/resources/lib/helper.py

```diff
@@ -0,0 +1,39 @@
+import putio
+import xbmcaddon
+
+__settings__ = xbmcaddon.Addon(id='plugin.video.putio')
+__lang__ = __settings__.getLocalizedString
+
+
+class PutioAuthFailureException(Exception):
+    """An authentication error occured."""
+
+    def __init__(self, header, message, duration=10000, icon='error.png'):
```

Comment:
instead of providing an own image, you could probably also just take xbmcgui.NOTIFICATION_ERROR . That way the error icon defined by the skin gets taken, which should look nicer in most cases.